packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.base.tsx

```diff
@@ -28,9 +28,13 @@ export class CheckboxBase extends BaseComponent<ICheckboxProps, ICheckboxState>
   constructor(props: ICheckboxProps, context?: any) {
     super(props, context);
 
-    this._warnMutuallyExclusive({
-      checked: 'defaultChecked'
-    });
+    initializeComponentRef(this);
+
+    if (typeof process !== 'undefined' && process.env.NODE_ENV !== 'production') {
```

Comment:
Orthogonal to this PR, but it feels like this check should be moved inside of `warnMutuallyExclusive` eventually

Reply:
I believe it is already, but... a higher level here means no empty function called or bundle impact.

Reply:
Makes even more sense!